docs/api/select.md

```diff
@@ -119,7 +119,7 @@ import RespondingToInteractionExample from '@site/static/usage/v7/select/basic/r
 
 When using objects for select values, it is possible for the identities of these objects to change if they are coming from a server or database, while the selected value's identity remains the same. For example, this can occur when an existing record with the desired object value is loaded into the select, but the newly retrieved select options now have different identities. This will result in the select appearing to have no value at all, even though the original selection in still intact.
 
-By default, the select uses object equality (`===`) to determine if an option is selected. This can be overridden by providing a property name or a function to the `compareWith` property.
```

Comment:
See discussion [here](https://github.com/ionic-team/ionic-framework/pull/28452#discussion_r1382018078) and [updated design doc](https://github.com/ionic-team/ionic-framework-design-documents/pull/182)